docs/crocks/Pred.md

```diff
@@ -2,12 +2,65 @@
 
 `Pred a`
 
---
+The `Pred` crock allows you to reuse the same predicate logic with different kind of inputs, the only think we ask of you is that you let `Pred` know how to get from your expected input to the predicate input.
```

Comment:
After this sentence can you add a sentence to let the user know their return type will be coerced to a boolean?